managed/services/telemetry/config.default.yml

```
@@ -600,6 +600,15 @@ telemetry:
       - metric_name: "pmm_server_node_os_version"
         label: "release"
 
+  - id: PMMNodeOSVersionInside
+    source: VM
+    query: node_os
+    summary: "PMM node OS version Inside"
```

```suggestion
    summary: "PMM node OS version"
```
I guess it's obvious, seems redundant :)

---

How are they different? The former is inside and the latter - outside?